src/librustc_driver/lib.rs

```diff
@@ -187,8 +187,9 @@ pub fn run_compiler(
         config
     };
 
+    let registry = Registry::new(&rustc_error_codes::DIAGNOSTICS);
```

Comment:
This seems to have lost the concern about not having the error codes from codegen.

Reply:
The concern was fixed in the PR which moved the error codes.